monkey/monkey_island/cc/ui/src/components/report-components/security/ScannedServers.js

```diff
@@ -32,17 +33,30 @@ class ScannedServersComponent extends React.Component {
   }
 
   render() {
+
     let defaultPageSize = this.props.data.length > pageSize ? pageSize : this.props.data.length;
     let showPagination = this.props.data.length > pageSize;
+
+    const howManyScannedMachines = this.props.data.length;
```

Comment:
Regarding readability I think it's better to name variables in a manner of `ScannedMachinesCount` instead of prepending `howMany`.

Reply:
Fixed